### DISCUSSION TITLE
Normal mode after pageup or pagedown

### DISCUSSION THREAD

**Discussion by aloispichler** [VSCodeVim/Vim#8263] [Q&A]
Hello,

I'd like VSCodeVim to return to _normal mode_ automatically after hitting _PageUp_ or _PageDown_ in insert mode.
How can this be accomplished?

**Comment by elzup** [comment]
I tried but `vimrc inoremap` way and `insertModeKeyBindings` way with `PageUp` didn't work. #6541

temporary solution

keybindings.json and use extension
https://marketplace.visualstudio.com/items?itemName=ryuta46.multi-command

```
{
  {
     "key": "[PageDown]",
     "command": "extension.multiCommand.execute",
     "args": {
       "sequence": ["extension.vim_escape", "scrollPageDown"]
     },
     "when": "editorTextFocus && vim.active && vim.mode == 'Insert'"
   },
   {
     "key": "[PageUp]",
     "command": "extension.multiCommand.execute",
     "args": {
       "sequence": ["extension.vim_escape", "scrollPageUp"]
     },
     "when": "editorTextFocus && vim.active && vim.mode == 'Insert'"
   }
 }
```
